lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -859,6 +864,18 @@ def _create_deployment_parameters(
             node.capability.set_extended_runbook(azure_node_runbook)
             nodes_parameters.append(azure_node_runbook)
 
+            if nodes_parameters[-1].enable_sriov is None:
```

Comment:
I think we should put this flag in AzureArmParameter level, and applies to each node. It's more clear.